frontend/mobilityai_react/App/Screens/PatientList.js

```diff
@@ -21,9 +21,10 @@ export default class PatientList extends React.Component {
         });
     }
     render() {
+        console.disableYellowBox = true;
```

Comment:
Warnings don't show up in prod version of the app.
https://facebook.github.io/react-native/docs/debugging.html#in-app-errors-and-warnings